ansible/roles/monitoring-statsd-fe-influxdb/templates/telegraf.conf.j2

```diff
@@ -114,7 +124,6 @@
       "nsq.*.topic.*.channel.* app.host.measurement.topic_name.measurement.channel_name.measurement*",
       "nsq.*.topic.* app.host.measurement.topic_name.measurement*",
       "nsq.* app.host.measurement*",
-      "kairos.* app.env.measurement*",
```

Comment:
We will need the "kairos.*" template as it has an "env" tag and is a special case like the nsq ones.